Cargo.toml

```
@@ -42,6 +41,11 @@ iota-stronghold = { git = "https://github.com/iotaledger/stronghold.rs", rev = "
 riker = { version = "0.4", optional = true }
 riker-patterns = { version = "0.4", optional = true }
 
+[dependencies.iota-crypto]
+git = "https://github.com/iotaledger/crypto.rs"
+rev = "c2514969c3f6693b86425ec84bada84b3ebea645"
+features = ["random", "sha", "pbkdf", "hmac", "sha", "bip39", "bip39-en", "chacha"]
```

```suggestion
features = ["random", "sha", "pbkdf", "hmac", "bip39", "bip39-en", "chacha"]
```
`"sha"` is listed twice here